packages/dashboard/src/pages/SetupPersona/Welcome/Article.tsx

```diff
@@ -97,30 +87,19 @@ export const Article: FC<Props> = ({ className, ...rest }) => {
 
     return (
         <article className={cx(classes.article, className)} {...rest}>
-            <Typography variant="h1" className={classes.h1}>
-                Help Us Improve Mask Network
-            </Typography>
-            <Typography className={classes.p} mt="35px">
+            <Typography className={classes.p}>
                 Mask Network aims to build an encrypted and decentralized social network, so you (all Internet users)
-                could send or browse encrypted posts with the 'Mask Network' extension or App. To improve our product,
-                we will need your help.
+                could send or browse encrypted posts with the 'Mask Network' extension or APP.
             </Typography>
-            <Typography variant="h2" className={classes.h2} mt="24px">
-                Mask Network never collects
+            <Typography className={classes.p}>
+                We provide services for digital encrypted identity and crypto wallet.
             </Typography>
-            <Typography className={classes.p} mt="24px">
-                Users' identity private keys, backup passwords, wallet payment passwords, wallet private keys, mnemonic
-                phrases, keystores.
-                <br />
-                Users' related information and browsing histories of their social accounts.
-                <br />
-                Users' addresses, balances, purchase histories, hash, or any personal information.
-                <br />
-                Users' complete IP address.
-                <br />
-                <br />
-                Mask Network keeps users' encrypted information in users'; browser, please keep it safe and make sure to
-                back up constantly.
+            <Typography className={classes.p}>
+                We are a browser extension that needs to request permission from certain websites in order to fulfill
+                our product's functions. For example, setting an NFT avatar in social media, searching for
+                cryptocurrency or artwork related to Twitter, and getting information enhancement on Twitter. Donating
+                to Gitcoin and voting in Snapshot on the Twitter timeline, and viewing information about assets and
+                artwork related to certain social accounts.
```

Comment:
```suggestion
Mask Network requires permission from certain websites in order to work.
For example,
setting up an NFT avatar on the SNS,
searching for cryptocurrencies or artworks related to Twitter,
adding enhancements on Twitter,
donating to Gitcoin,
voting on Snapshot in the Twitter timeline,
and viewing information on assets and artworks related to specific accounts.
```

A list might be better. You're listing so many things.

Reply:
Same as above.